static/install

```diff
@@ -267,7 +284,7 @@ bash)
 
     for bash_config in $bash_configs; do
         # deliberately omit the home directory from the pattern so "~" and "$HOME" can be used interchangeably
-        if [ -f "$bash_config" ] && match=$(grep -n ".dune/env/env.bash" "$bash_config"); then
+        if [ -f "$bash_config" ] && match=$(grep -n ".local/share/dune/env/env.bash" "$bash_config"); then
```

Comment:
Why not use `remove_home` here?

Reply:
I accidentally missed the change. Updated it now. Thanks.